topostats/validation.py

```diff
@@ -133,7 +133,7 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
         },
         "grains": {
             "run": Or(True, False, error="Invalid value in config for grains.run, valid values are 'True' or 'False'"),
-            "absolute_smallest_grain_size": int,
+            "smallest_grain_size": int,
```

Comment:
Would need updating if variable name changed, could perhaps check that its positive

```suggestion
            "smallest_grain_size_nm": lambda n: n > 0.0,
```